schema/deploy/shipping/views.sql

```diff
@@ -59,7 +59,9 @@ create or replace view shipping.reportable_condition_v1 as
                                    'collections-kiosks',
                                    'collections-self-test',
                                    'collections-swab&send-asymptomatic',
-                                   'collections-kiosks-asymptomatic')
+                                   'collections-kiosks-asymptomatic',
+                                   'collections-environmental')
+    and coalesce(encountered::date, date_or_null(sample.details ->> 'date')) >= '2020-01-01'
```

Comment:
Looks like `collections-environmental` slipped into this commit about the date filter.

Reply:
Thanks for catching that